actions/telegram-send/exo.action.yaml

```diff
@@ -0,0 +1,20 @@
+name: telegram-send
+description: Bots telegram
```

Comment:
This is not a sentence. Try `description: Send a message through a Telegram bot`